spec/system/post_event_spec.rb

```diff
@@ -92,4 +94,29 @@
     expect(page.find(".d-modal")).to have_text("test_group")
     expect(page.find(".d-modal .custom-field-input").value).to eq("custom value")
   end
+
+  context "when using bulk inline invite" do
+    let!(:post) do
+      PostCreator.create(
+        admin,
+        title: "My test meetup event",
+        raw: "[event name='cool-event' status='public' start='2222-02-22 00:00' ]\n[/event]",
+      )
+    end
+
+    fab!(:invitable_user_1) { Fabricate(:user) }
+    fab!(:invitable_user_2) { Fabricate(:user) }
+
+    it "can invite users to an event" do
+      visit(post.topic.url)
+
+      post_event_page.open_bulk_invite_modal
+      bulk_invite_modal_page.set_invitee_at_row(invitable_user_1.username, "going", 1)
+      bulk_invite_modal_page.add_invitee
+      bulk_invite_modal_page.set_invitee_at_row(invitable_user_2.username, "not_going", 2)
+      bulk_invite_modal_page.send_invites
```

Comment:
would something like this work?
```suggestion
      bulk_invite_modal_page
        .set_invitee_at_row(invitable_user_1.username, "going", 1)
        .add_invitee
        .set_invitee_at_row(invitable_user_2.username, "not_going", 2)
        .send_invites
```

Reply:
yes but Im mixed on this being really better

Reply:
Ok I did, because I don't know... let's say it's better :+1:

Reply:
I ❤️ the chained version slightly more than the procedural one 👍

Reply:
I feel that has less _noise_